frontend/pulse3d/src/pages/_app.js

```diff
@@ -48,11 +48,13 @@ function Pulse({ Component, pageProps }) {
       navigator.serviceWorker.addEventListener("message", ({ data }) => {
         // data returned is a boolean if auth tokens are present. Otherwise return user to login
         // might need auth check to include actual fetch request in SW to check token status if this becomes a problem
+        if (data.logout !== true) {
+          return;
+        }
```

Comment:
do we need `if (logout) setLoggedOutAlert(true);` ?

Reply:
No I just tested it without it. I think this is left from my debug. I removed it.